core/schema/src/main/resources/org/hawkular/schema/bootstrap.groovy

```diff
@@ -48,8 +48,14 @@ if (reset) {
   executeCQL("DROP KEYSPACE IF EXISTS $keyspace", 20000)
 }
 
-executeCQL("CREATE KEYSPACE IF NOT EXISTS $keyspace WITH replication = {'class': 'SimpleStrategy', " +
-    "'replication_factor': $replicationFactor}")
+def keyspaceCreate = "CREATE KEYSPACE IF NOT EXISTS $keyspace WITH replication = {'class': 'SimpleStrategy', " +
+    "'replication_factor': $replicationFactor}"
+
+if (keyspace.equals("hawkulartest") || keyspace.equals("hawkular_metrics_rest_tests")) {
```

Comment:
Would it be better to make this a general configuration? Right now you it is very specific to two keyspaces, I would make this configurable via flag and not based on the keyspace name.